### PR TITLE
Define react-redux 7.1 hooks API

### PR DESCRIPTION
**Pull request by halitogunc**
#62 
see: https://react-redux.js.org/api

@dbo please

### CODE REVIEW

File: npm/react-redux/7.1/@n4jsd/react-redux/lib/index.n4jsd [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+     * If true, implements shouldComponentUpdate and shallowly compares the result of mergeProps,
+     * preventing unnecessary updates, assuming that the component is a “pure” component
+     * and does not rely on any input or state other than its props and the selected Redux store’s state.
+     * 
+     * @default true
+     */
+    public get pure?(): boolean;
+    
+    /**
+     * If true, uses React's forwardRef to expose a ref of the wrapped component
```
Comment (dbo) [inline npm/react-redux/7.1/@n4jsd/react-redux/lib/index.n4jsd]: trailing .

**Review by dbo — APPROVED** [review]
👍

File: npm/react-redux/7.1/@n4jsd/react-redux/lib/index.n4jsd (hunk 1 above)
Comment (halitogunc) [reply npm/react-redux/7.1/@n4jsd/react-redux/lib/index.n4jsd]: 👍

**Comment by dbo** [comment]
on master

**Pull request merged**